### ISSUE TITLE
Image optimising issue with PNGQuant. Colour shift in Firefox.

### ISSUE THREAD

**Issue by petemcbide** [bdkjones/CodeKit#445]
**Quick, short summary:**
PNG Optimizer (using PNGQuant with default settings) seems to do something with the colour profile of an image, that only affects the colour displayed in Firefox.

**Expected results:**
Previewed a page in Safari, Chrome and Firefox in which an image is placed on a background that is the same colour as the image background. Expecting the image to blend perfectly with the background colour.

**Actual results:**
Image and background element match perfectly in Chrome and Safari, there is a colour shift in Firefox.

**Exact steps to reproduce:**
Optimize an image using Codekit PNGQuant, default settings.

**A link to download a simplified project or file that shows the issue:**
Simple HTML doc and screenshot that shows the issue: https://www.dropbox.com/s/eteu857vo674uaq/colour.zip?dl=0

**Your configuration (any details about your system that you think might be relevant)**
Latest MacOS and latest browsers.

**Comment by bdkjones** [comment]
Interesting. Could you open an issue on the PNGQuant repo with the original PNG and the optimized one? Tell them that the options being passed into PNGQuant are:

```
--strip
--speed=1
--quality=0-[whatever quality you have selected in CodeKit, as an integer]
```

The version of PNGQuant is in CodeKit's About window; be sure to post that as well. Thanks!

**Comment by petemcbide** [comment]
Sure can – https://github.com/kornelski/pngquant/issues/292

**Comment by bdkjones** [comment]
I doubt the two are related. Can you provide a test case for the JPEG issue?

**Comment by BenjaminHoegh** [comment]
**Before**
![ronaturmennesker](https://user-images.githubusercontent.com/22152591/37530304-82fa11ac-2939-11e8-89f3-0de3150a0fe7.jpg)

After
![ronaturmennesker-1](https://user-images.githubusercontent.com/22152591/37530305-8317526c-2939-11e8-9c6f-97f8b41bb4fd.jpg)


please note there are copyright on the images so don't try to copy

**Comment by BenjaminHoegh** [comment]
@bdkjones Just the images or a log or something??

**Comment by bdkjones** [comment]
The JPEG issue will be resolved in 3.6. I've changed the optimizer to retain ICC color profiles instead of discarding them.
